### DISCUSSION TITLE
Streamline Icons

### DISCUSSION THREAD

**Discussion by sharfatehi** [twilio-labs/paste#1610] [Q&A] [answered]
Hello! Where can I get access to all of the streamline icons? I am currently working on swapping out the current Flex icons with streamline, and there are only a handful available in Paste (unless I am mistaken). 

Thanks!

**Comment by richbachman** [comment, marked as answer]
Hi @sharfatehi. You'll need to reach out to the Brand team (#mkg-brand on Slack) in order to get access to Streamline. 

We're also curious which icons you're looking for. Let us know. Thanks!

**Reply by sharfatehi** [reply]
Thanks @richbachman ! I'll do that :) 

I'm working on swapping ALL the existing ones out with 'like' ones in streamline. 
To verify, paste is using all streamline icons, is that correct?

**Reply by richbachman** [reply]
Yeah. for the most part, we use streamline icons.
